src/ripple/overlay/impl/Message.cpp

```diff
@@ -26,7 +26,7 @@ namespace ripple {
 
 Message::Message (::google::protobuf::Message const& message, int type)
 {
-    unsigned const messageBytes = message.ByteSize ();
```

Comment:
type is actually `size_t`, so maybe just switch to `auto` here? ( I know you were just fixing a deprecated method call...)

Reply:
switched to auto